sdk/index.ts

```diff
@@ -49,9 +53,10 @@ interface FortaConfig {
   keyfilePassword?: string
 }
 
-type Initialize = () => Promise<void>
+type Initialize = () => Promise<InitializeResponse>
```

Comment:
hmmm this could cause type errors for existing bots.

also, what would a bot return if they are not using InitializeResponse (but are using initialize to fetch some data)